src/styles/blocks/stars.css

```diff
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: var(--active-star);
+}
+
+.stars--2 :nth-child(-n + 2) {
```

Comment:
Remove the space between `.stars--2` and `:nth-child(-n + 2)` to correctly target the first two child elements directly inside `.stars--2`.